src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1256,6 +1264,37 @@ public function getExpressionLanguageProviders()
     }
 
     /**
+     * Sets a definition template to automatically apply to all Definition that opt into it.
+     *
+     * @param string $interface The class or interface to match
+     * @param $childDefinition ChildDefinition
+     */
+    public function addAutomaticInstanceofDefinition($interface, ChildDefinition $childDefinition)
```

Comment:
could this return the ChildDefinition instead of this second arg?
that would allow things like `$container->addAutomaticInstanceofDefinition(\Twig_ExtensionInterface::class)->addTag('twig.extension')`